package.json

```diff
@@ -28,18 +30,53 @@
     "url": "https://github.com/silverstripe/silverstripe-cms/issues"
   },
   "homepage": "https://github.com/silverstripe/silverstripe-cms#readme",
-  "devDependencies": {
-    "@silverstripe/webpack-config": "^0.2.5",
-    "babel-core": "^6.7.4",
+  "dependencies": {
+    "@silverstripe/webpack-config": "^0.3.0",
+    "babel-jest": "^20.0.3",
     "babel-loader": "^7.0.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-es2016": "^6.24.1",
-    "babel-preset-react": "^6.5.0",
-    "eslint": "^2.7.0",
-    "eslint-config-airbnb": "^6.2.0",
-    "eslint-plugin-react": "^4.3.0"
+    "classnames": "^2.2.5",
+    "deep-freeze-strict": "^1.1.1",
+    "jest-cli": "^19.0.2",
```

Comment:
`@silverstripe/webpack-config`, `babel-jest` and `jest-cli` should be devDependencies